plugins/chat/spec/system/deleted_message_spec.rb

```diff
@@ -18,9 +17,10 @@
     it "shows as deleted" do
       chat_page.visit_channel(channel_1)
       expect(channel_page).to have_no_loading_skeleton
-
-      channel_page.expand_message_actions(message_1)
-      find("[data-value='deleteMessage']").click
+      channel_page.send_message("aaaaaaaaaaaaaaaaaaaa")
+      expect(page).to have_no_css("[data-staged-id]")
+      last_message = find(".chat-message-container:last-child")
+      channel_page.delete_message(OpenStruct.new(id: last_message["data-id"]))
```

Comment:
This might seem slightly hacky, but I think this is a nice way to have an the API always accept a message as long a it quacks like a message without building something too complex or having a lot of methods similar to `hover_by_message_or_id`

Reply:
I agree with this 👍